packages/analytics/amplify_analytics_pinpoint/example/backend/lib/backend-stack.ts

```diff
@@ -0,0 +1,244 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// A copy of the License is located at
+//
+//  http://aws.amazon.com/apache2.0
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+import * as cdk from "aws-cdk-lib";
+import { Construct } from "constructs";
+import * as cognito from "aws-cdk-lib/aws-cognito";
+import * as pinpoint from "aws-cdk-lib/aws-pinpoint";
+import * as kinesis from "aws-cdk-lib/aws-kinesis";
+import * as iam from "aws-cdk-lib/aws-iam";
+import * as lambda from "aws-cdk-lib/aws-lambda";
+import * as lambda_nodejs from "aws-cdk-lib/aws-lambda-nodejs";
+import * as lambda_events from "aws-cdk-lib/aws-lambda-event-sources";
+import * as appsync from "@aws-cdk/aws-appsync-alpha";
+import * as dynamodb from "aws-cdk-lib/aws-dynamodb";
+import * as path from "path";
+import { CfnOutput, Duration, Expiration, RemovalPolicy } from "aws-cdk-lib";
+
+export class BackendStack extends cdk.Stack {
```

Comment:
request: Could this stack also include an s3 bucket with name that matches this pattern https://github.com/aws-amplify/amplify-ci-support/blob/main/src/integ_test_resources/flutter/amplify/cloudformation_template.yaml#L32 (starts with "amplify-test") and then have the provision script upload the amplifyconfig file it creates to that bucket? That way, someone else (or the CI) can pull it down as long as we later add a pull script. If the bucket has that name pattern then existing gh action role should have no issue. I'm also open to other ideas to solve that problem but would like for it to be part of this stack.

Reply:
Sure - would I just write the amplifyconfiguration.dart to the root of the bucket?

Reply:
That's what I was thinking. The pull script (not needed in this PR) can just use aws CLI command to download.